backend/app/main.py

```
@@ -580,6 +580,26 @@ async def log_requests(request: Request, call_next):
+        )
+
+        # Mount system MCP server at /mcp/system
+        system_mcp_app = _create_system_mcp_app()
+        app.mount("/mcp/system", system_mcp_app)
+        logger.info("System MCP server mounted at /mcp/system")
+
+        # Mount knowledge MCP server at /mcp/knowledge
+        knowledge_mcp_app = _create_knowledge_mcp_app()
+        app.mount("/mcp/knowledge", knowledge_mcp_app)
+        logger.info("Knowledge MCP server mounted at /mcp/knowledge")
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)